fastcrypto-zkp/src/bn254/zklogin_test_vectors.json

```diff
@@ -40,5 +40,11 @@
     "kid": "6d361dc9637a275eb585a915af26198ff0d97326ca13f4baf0e4805f72f2a9a0",
     "n": "y_8hHwq7w2yE4968sbQF98iGUhnu0BwyB5khTxVPAcUnMCYdp61zYcRWml2zdY4HAfq-Nnjb_pAli6I66Vpe9IE8Gf8uGRB0oYIo2S6tYMEe0lhRaEDYVbMdQkuKxTIYMNBXSd_kCHKJM1ZUAo7uFoq_bWuzt2hRG2-79z-Ycbiw0wil0rzFHlpNBKsBLKM4GSGUwOejaL2zCiE_rjf77AvOaJLRd4I_DBYG16t8D1BkxbhkcQCmOxYGG0NqjP3z0lz-w1ALqHCNfhzczZOsgaCrbSlcTKcBTq1syAUUhQmounW7nG5clBIfPQRVH7jCoPztiJUZg6Xz1AN6V07xnw",
     "provider": "Threedos"
+  },
+  {
+    "jwt": "eyJ0eXAiOiJqd3QiLCJhbGciOiJSUzI1NiIsImtpZCI6Ik81cnl4Ri16TUNMbVM2aFFoY1RDM3BBQWhRNFlZUEVIb2lRdDFxeF84Nm8ifQ.eyJpc3MiOiJodHRwczovL2FjY291bnRzLmZhbnR2LndvcmxkIiwiYXpwIjoicjI0YnNreHlhZnd3dWE2OGV0MndtdXFleW9hLmFwcHMuZmFudHYud29ybGQiLCJhdWQiOiJyMjRic2t4eWFmd3d1YTY4ZXQyd211cWV5b2EuYXBwcy5mYW50di53b3JsZCIsInN1YiI6IjY0NDM1MDVjNWRlNTkzNWRhZjE1NjM1YyIsIm5vbmNlIjoiZjR3eXRnYmkzNGpna2VmaGp3ZXIxMTIxMjEiLCJpYXQiOjE3MjMxOTI5MDAsImV4cCI6MTcyMzE5NjUwMCwianRpIjoibHptZ2s3dGgtY2VpdDRvNW1kbyJ9.DbC5_2tRWhEcaj8GaWEu7aRe9iH3sKHRVwPPSDSfpU83lRcqCkbPh9Glzcv-lAie0xenhU_jeFP9qeBZ0wKTnaJonhbrzYDDrGwjBEOqPqh5XE5BGM6qBqLSZJmmvJ7KWqwb7IHhXOGCAJFwdpMoZ7mWxIw9xqcZbDhi8w4dDMVr3lMsDTIhy8x_QnzDqf1gfx9ZgDlWY3CelKK67J2mtiW0IfaEPGq5ITUQxE8sMfDfgqmsdGiIJFHG9xdtVjJWMI--oBmfwD97byyu3Ss_YyBxjizzP6cAE2G3tb7ERpPlRX84qdd0uQUOCalQxBfaIiAnea6GEzAHIJWFrd1AtA",
```

Comment:
the nonce for this should be hTPpgF7XAKbW37rEUS6pEVZqmoI

Reply:
Here is the amended JWT : https://github.com/MystenLabs/fastcrypto/pull/830/commits/13ff461ac7ada957ab6c290b5a8fc93db835aaa5